macos/Classes/FlutterSecureStoragePlugin.m

```diff
@@ -0,0 +1,201 @@
+#import "FlutterSecureStoragePlugin.h"
```

Comment:
It seems pretty close (if not 100% same) to iOS version, can it be reused aka DRY?

Reply:
[package_info](https://github.com/flutter/plugins/blob/master/packages/package_info) implements an approach for making code dry between iOS and macOS.

Reply:
well, i didn’t know that flutter plugins can handle symblinks. It is a good approach though

Reply:
yeah, I just saw it today.

Reply:
If the code is consistent with iOS, then this only requires the ln -s command. Of course, we can also share most of the code and do some meticulous work by judging the platform in OC.

I don't understand OC code, so please help to finish this work as soon as possible, thank you.